README.md

```diff
@@ -123,6 +123,9 @@ specify any custom user configuration arguments.
 --clean-git-repo
 clean out any 'bad' local git repo you already have.
 
+--create-debug-symbols-package
+create a debug-symbols only archive if debug symbols are generated.
```

Comment:
I think this needs a slight change in grammar.  Perhaps:

`create a debug-symbols only archive (if debug symbols were generated).`